walletlibrary/src/main/java/com/microsoft/walletlibrary/util/http/httpagent/IResponse.kt

```diff
@@ -57,7 +58,7 @@ class IResponse(
         }
         return NetworkingException(
             message = message,
-            code = code,
+            code = VerifiedIdExceptions.NETWORKING_EXCEPTION.value,
             correlationId = correlationId,
             statusCode = status.toString(),
```

Comment:
This may need to be `code.toString()`

Reply:
We have two different codes for Networking exception. statusCode is the http error code. code is from the enum of error codes for WL.